lib/builders/latexmk.js

```diff
@@ -77,12 +80,35 @@ export default class LatexmkBuilder extends Builder {
 
     if (engine.match(latexRegExp)) {
       args.push(`-latex="${engine}"`)
-      if (outputFormat !== 'dvi') {
-        atom.notifications.addWarning(
-          'PS/PDF output is not supported by LaTeX engines.'
-        )
+      if (outputFormat === 'pdf') {
+        switch (producer) {
+          case 'default':
+            // default is an option to protect a user's .latexmkrc from override
+            args.push('-pdfdvi')
+            break
+          case 'dvipdf':
+            args.push('-pdfdvi')
+            args.push('-e "\\$dvipdf = \'dvipdf %O %S %D\';"')
+            break
+          case 'dvipdfm':
```

Comment:
As `dvipdfm` is a symlink to `dvipdfmx` since TeX Live 2009 and the driver file for `graphicx` package is removed in TL 2013, dvipdfm is regarded as obsolete for quite long. Details about compatibility between dvipdfm and dvipdfmx are a bit intricate, but in short, the use of dvipdfm with CJK documents is strongly discouraged unless under very limited circumstances. Hence, I think support for dvipdfm is an excessive burden that doesn't need to be imposed. I'm just letting you know because it's entirely up to you to decide whether or not to support such outdated distributions.